turbo.json

```diff
@@ -5,13 +5,22 @@
       "dependsOn": ["^build", "gen"],
       "outputs": ["dist", "dist-cli"],
       "env": [
-        "SENTRY_AUTH_TOKEN", 
-        "SENTRY_ORG", 
+        "SENTRY_AUTH_TOKEN",
+        "SENTRY_ORG",
         "SENTRY_PROJECT",
         "LANGFUSE_PUBLIC_KEY",
         "LANGFUSE_SECRET_KEY",
         "LANGFUSE_BASE_URL",
-        "NEXT_PUBLIC_ENV_NAME"
+        "NEXT_PUBLIC_ENV_NAME",
+        "TRIGGER_ACCESS_TOKEN",
+        "TRIGGER_PROJECT_ID"
+      ]
+    },
+    "@liam-hq/jobs#deploy": {
+      "dependsOn": ["^build"],
+      "passThroughEnv": [
+        "TRIGGER_ACCESS_TOKEN",
+        "TRIGGER_PROJECT_ID"
       ]
```

Comment:
This is how to write to pass environment variables to the package in turborepo.
ref: https://turbo.build/repo/docs/reference/configuration#passthroughenv